airbyte-cdk/python/airbyte_cdk/sources/file_based/file_based_source.py

```diff
@@ -298,3 +309,10 @@ def _validate_and_get_validation_policy(self, stream_config: FileBasedStreamConf
     def _validate_input_schema(self, stream_config: FileBasedStreamConfig) -> None:
         if stream_config.schemaless and stream_config.input_schema:
             raise ValidationError("`input_schema` and `schemaless` options cannot both be set", model=FileBasedStreamConfig)
+
+    @staticmethod
+    def _use_file_transfer(parsed_config: AbstractFileBasedSpec) -> bool:
+        use_file_transfer = (
+            hasattr(parsed_config.delivery_method, "delivery_type") and parsed_config.delivery_method.delivery_type == "use_file_transfer"
+        )
+        return use_file_transfer or parsed_config.use_file_transfer
```

Comment:
This is a nice helper. I appreciate that the business logic can live here and just return a boolean. 👍